src/resolvers/resolverUtils.ts

```diff
@@ -347,15 +351,20 @@ export function prepareCourseCapacities({
   return combinedRequest;
 }
 
-function courseSectionInputToCourse(course: CourseSectionInput) {
+async function courseSectionInputToCourse(
+  course: CourseSectionInput
+): Promise<CourseType> {
+  const user = await findUserByUsername(course.professors[0]);
```

Comment:
this will only grab the first professor? is it because the professor field has 1 user only?

Reply:
Yes, the professor field is presumed to have 1 user only. It was discussed earlier with FE (I think Ken raised that topic).

Reply:
https://discord.com/channels/973344930699313203/973345629914947675/998728216825045023